vkcs/internal/services/containerinfra/v1/clusters/requests.go

```diff
@@ -31,10 +31,12 @@ type CreateOpts struct {
 	APILBFIP                  string            `json:"api_lb_fip,omitempty"`
 	RegistryAuthPassword      string            `json:"registry_auth_password,omitempty"`
 	AvailabilityZone          string            `json:"availability_zone,omitempty"`
+	AvailabilityZones         []string          `json:"availability_zones,omitempty"`
 	LoadbalancerSubnetID      string            `json:"loadbalancer_subnet_id,omitempty"`
 	InsecureRegistries        []string          `json:"insecure_registries,omitempty"`
 	DNSDomain                 string            `json:"dns_domain,omitempty"`
 	SecurityPolicySyncEnabled *bool             `json:"security_policy_sync_enabled,omitempty"`
+	ClusterType               string            `json:"cluster_type"`
```

Comment:
I set the default value in terraformer as standard, so cluster_type will always be specified. So if the default cluster in the cloud is changed, the user will still get a cluster that puts the default in the tf and this will not be a surprise for him.

Reply:
Yeah, default value is good, but you should always consider all options when working with different parts of code, and don't rely on some logic somewhere. It's just a body of an API request, and should depend only on API spec.